src/content/app/genome-browser/state/track-panel/trackPanelSlice.ts

```diff
@@ -284,3 +334,46 @@ export const updateCollapsedTrackIds =
       })
     );
   };
+
+const trackPanelSlice = createSlice({
+  name: 'genome-browser-track-panel',
+  initialState: getTrackPanelState() as TrackPanelState,
+  reducers: {
+    setTrackPanelDataFromUrl(state, action: PayloadAction<ParsedUrlPayload>) {
+      const { activeGenomeId } = action.payload;
+      if (!state[activeGenomeId]) {
+        state.activeGenomeId = getTrackPanelStateForGenome(activeGenomeId);
+      }
+    },
```

Comment:
Why does trackPanelSlice need to keep another copy of `activeGenomeId`? Especially since the `updateTrackPanelForGenome` reducer receives an action that contains a genome id.

Reply:
It supposed to be `state[activeGenomeId]`! I've updated it.